packages/documentation/pages/utilities.ts

```diff
@@ -42,6 +46,7 @@ export type ComponentValue = {
 	defaultSlot: string | null
 	hasActions: boolean
 	hasHelpTextSlot: boolean
+	hasRemoteUpload: boolean
```

Comment:
is this still needed?

Reply:
It's still needed to generate the component's code when calling `generateComponentCode` from `form-fiels.vue` (it's similar to hasActions for KtFieldSelects).